pkg/agent/openflow/client.go

```diff
@@ -1010,7 +1018,7 @@ func (c *client) InstallSNATMarkFlows(snatIP net.IP, mark uint32) error {
 	cacheKey := fmt.Sprintf("s%x", mark)
 	c.replayMutex.RLock()
 	defer c.replayMutex.RUnlock()
-	return c.addFlows(c.featureEgress.cachedFlows, cacheKey, []binding.Flow{flow})
+	return c.modifyFlows(c.featureEgress.cachedFlows, cacheKey, []binding.Flow{flow})
```

Comment:
why is the change?

Reply:
After decoupling snatflow and qosFlow, changed back.